cmd/lukso/init.go

```diff
@@ -9,15 +9,25 @@ import (
 
 // initializeDirectory initializes a working directory for lukso node, with all configurations for all networks
 func initializeDirectory(ctx *cli.Context) error {
+	if cfg.Exists() {
+		message := "This folder has been already initialized. Do you want to re-initialize? Please note that configs in this folder will NOT be overwritten [Y/n]:\n> "
+		input := registerInputWithMessage(message)
+		if !strings.EqualFold(input, "y") && input != "" {
+			return nil
+		}
+	}
+
 	for _, dependency := range clientDependencies {
 		// this logic may fail when folder structure changes, but this shouldn't be the case
-		if !strings.Contains(dependency.filePath, "./config") {
+		if !strings.Contains(dependency.filePath, "./configs") {
```

Comment:
you can reuse the "path" global var here also - cf other comments

Reply:
I changed this check entirely, now that dependencies have their `isBinary` field